prime-dotnet-webapi/Controllers/EnrolleeAgreementsController.cs

```diff
@@ -243,5 +243,32 @@ public async Task<ActionResult> GetAccessTermSignable(int enrolleeId, int agreem
 
             return Ok(download);
         }
+
+        // GET: api/enrollees/5/agreements/current/obo-to-ru
+        /// <summary>
+        /// Gets boolean re: whether enrollee's current agreement type is OBO and the agreement type
+        /// they would be assigned is RU, if automatic assignment occurred today
+        /// </summary>
+        /// <param name="enrolleeId"></param>
+        [HttpGet("{enrolleeId}/agreements/current/obo-to-ru", Name = nameof(IsOboToRuAgreementTypeChange))]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(ApiResultResponse<bool>), StatusCodes.Status200OK)]
+        public async Task<ActionResult> IsOboToRuAgreementTypeChange(int enrolleeId)
+        {
+            var record = await _enrolleeService.GetPermissionsRecordAsync(enrolleeId);
+            if (record == null)
+            {
+                return NotFound($"Enrollee not found with id {enrolleeId}");
+            }
+            if (!record.AccessableBy(User))
```

Comment:
Since this is only being used in the enrollee side of things, we can avoid doing this check and add
``` [Authorize(Roles = Roles.PrimeEnrollee)] ```
to the method

Reply:
I think this `if (!record.AccessableBy(User))` check is still necessary so that **another** Enrollee cannot check the OBO to RU state of an Enrollee's agreement, however benign that may be.

Reply:
Yeah, that is my bad. Was on the wrong tangent!